src/open_inwoner/scss/components/KCMSurvey/KCMSurvey.scss

```diff
@@ -0,0 +1,13 @@
+.kcm-survey {
+  position: fixed;
+  top: 40%;
```

Comment:
Vertical positioning appears to be tricky since the thing is rotated. I note that the element is displayed slightly below the center of the screen.

Reply:
Yeah this is quite tricky to make perfect, so I feel like this is pretty close? If perfection is needed, I would remove the style from the 'kcm-survey' class and add the positioning to the link-styles (again positioning only) from the https://gemeente.groningen.nl/ "Uw mening" anchor link ;-) and then correct for our own padding and colors etc.

Reply:
I'll take the styling from Groningen :+1: